Applications/DataExplorer/mainwindow.cpp

```diff
@@ -839,48 +837,33 @@ void MainWindow::about()
 
 QMenu* MainWindow::createImportFilesMenu()
 {
-    auto* signal_mapper = new QSignalMapper(this);  // owned by MainWindow
     QMenu* importFiles = new QMenu("&Import Files", this);
-    QAction* feflowFiles = importFiles->addAction("&FEFLOW Files...");
-    connect(feflowFiles, SIGNAL(triggered()), signal_mapper, SLOT(map()));
-    signal_mapper->setMapping(feflowFiles, ImportFileType::FEFLOW);
-    QAction* gmsFiles = importFiles->addAction("G&MS Files...");
-    connect(gmsFiles, SIGNAL(triggered()), signal_mapper, SLOT(map()));
-    signal_mapper->setMapping(gmsFiles, ImportFileType::GMS);
-    QAction* gmshFiles = importFiles->addAction("&GMSH Files...");
-    connect(gmshFiles, SIGNAL(triggered()), signal_mapper, SLOT(map()));
-    signal_mapper->setMapping(gmshFiles, ImportFileType::GMSH);
-    QAction* gocadTsFiles = importFiles->addAction("&Gocad TSurface...");
-    connect(gocadTsFiles, SIGNAL(triggered()), signal_mapper, SLOT(map()));
-    signal_mapper->setMapping(gocadTsFiles, ImportFileType::GOCAD_TSURF);
+    importFiles->addAction("&FEFLOW Files...",
+                           [=] { open(ImportFileType::FEFLOW); });
+    importFiles->addAction("G&MS Files...",
+                           [=] { open(ImportFileType::GMS); });
+    importFiles->addAction("&GMSH Files...",
+                           [=] { open(ImportFileType::GMSH); });
+    importFiles->addAction("&Gocad TSurface...",
+                           [=] { open(ImportFileType::GOCAD_TSURF); });
 #ifdef OGS_USE_NETCDF
-    QAction* netcdfFiles = importFiles->addAction("&NetCDF Files...");
-    connect(netcdfFiles, SIGNAL(triggered()), signal_mapper, SLOT(map()));
-    signal_mapper->setMapping(netcdfFiles, ImportFileType::NETCDF);
+    importFiles->addAction("&NetCDF Files...",
+                           [=] { open(ImportFileType::NETCDF); });
 #endif  // OGS_USE_NETCDF
-    QAction* petrelFiles = importFiles->addAction("&Petrel Files...");
-    connect(petrelFiles, SIGNAL(triggered()), this, SLOT(loadPetrelFiles()));
-    QAction* rasterFiles = importFiles->addAction("&Raster Files...");
-    connect(rasterFiles, SIGNAL(triggered()), signal_mapper, SLOT(map()));
-    signal_mapper->setMapping(rasterFiles, ImportFileType::RASTER);
+    importFiles->addAction("&Petrel Files...",
+                           [=] { loadPetrelFiles(); });
+    importFiles->addAction("&Raster Files...",
+                           [=] { open(ImportFileType::RASTER); });
 #if defined VTKFBXCONVERTER_FOUND
-    QAction* rasterPolyFiles = importFiles->addAction("R&aster Files as PolyData...");
-    connect(rasterPolyFiles, SIGNAL(triggered()), this, SLOT(map()));
-    _signal_mapper->setMapping(rasterPolyFiles, ImportFileType::POLYRASTER);
+    importFiles->addAction("R&aster Files as PolyData...",
```

Comment:
@rinkk Is the placement of '&' correct or should it be
```suggestion
    importFiles->addAction("&Raster Files as PolyData...",
```

Reply:
```R&aster``` is correct here.